packages/js/product-editor/src/index.ts

```diff
@@ -40,6 +40,10 @@ export { PostTypeContext } from './contexts/post-type-context';
  */
 export * from './products';
 
+export { default as SiteHub } from './products-app/site-hub';
+export { default as SidebarContent } from './products-app/sidebar';
+export { unlock } from './lock-unlock';
```

Comment:
In a follow up, I'd like to move these to `@woocommerce/admin-layout` so settings and product-editor can consume from the same place.

Reply:
That's a good idea. Do we have an existing issue created for this already?

Reply:
Created here https://github.com/woocommerce/woocommerce/issues/52520